metafix/src/main/java/org/metafacture/metafix/FixPath.java

```diff
@@ -60,6 +60,7 @@ private FixPath(final String[] path) {
     }
 
     /*package-private*/ Value findIn(final Array array) {
+
```

Comment:
I didn't suggest that.

Reply:
I thought that was implied by:

> Empty lines at start and end of a class body should be consistent; either both (usually) or none (for short bodies).

And:

> (another example would be FixPath.findIn() in this pull request)

Anyway, please add blank lines as you like, I have no particular preference.

Reply:
> I thought that was implied by:
> 
> > Empty lines at start and end of a class body should be consistent; either both (usually) or none (for short bodies).

This referred to _class bodies_.

> And:
> 
> > (another example would be FixPath.findIn() in this pull request)

We were talking about empty lines in general at this point. What I meant was:

```diff
     /*package-private*/ Value findIn(final Hash hash) {
         final String currentSegment = path[0];
         final FixPath remainingPath = new FixPath(tail(path));
-
         if (currentSegment.equals(ASTERISK)) {
             // TODO: search in all elements of value.asHash()?
             return remainingPath.findIn(hash);
         }
-
         final Value value = hash.get(currentSegment);
         return value == null || path.length == 1 ? value : value.extractType((m, c) -> m
                 .ifArray(a -> c.accept(remainingPath.findIn(a)))
```

> Anyway, please add blank lines as you like, I have no particular preference.

Okay, will do (as I come across them).